plugins/3MFWriter/ThreeMFWriter.py

```diff
@@ -268,6 +269,10 @@ def _getMaterialPackageMetadata() -> List[Dict[str, str]]:
                 # Don't export materials not in use
                 continue
 
+            if type(extruder.material) is EmptyInstanceContainer:
```

Comment:
```suggestion
           if isinstance(container, type(ContainerRegistry.getInstance().getEmptyInstanceContainer())):
```

In other places we do it like this, so i think it makes sense to use the same way of doing it.